Misc/NEWS.d/next/Core and Builtins/2019-05-05-18-09-40.bpo-34556.o9kfpu.rst

```diff
@@ -0,0 +1 @@
+Add --upgrade-deps to venv module
```

Comment:
Use the ReST markup like `--upgrade-deps`. Add "Patch by ..."

Reply:
Sure thing.